{
  "api": "index",
  "file": "/test/free/data_stream/260_logs_failure_store.yml",
  "name": "Test logs-*-* data streams have failure store enabled by default",
  "origin": "yaml",
  "request": {
    "args": {
      "body": {
        "@timestamp": "2023-01-01T12:01:00Z",
        "host": {
          "name": "server-02"
        },
        "message": {
          "struct": {
            "value": 42
          }
        },
        "severity": "ERROR"
      },
      "index": "logs-app-default",
      "refresh": true
    }
  },
  "response": {
    "headers": {
      "content-length": "235",
      "content-type": "application/json",
      "location": "/.fs-logs-app-default-2025.10.31-000002/_doc/b5rEOZoBCWI_1WX5p0d2",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "_id": "b5rEOZoBCWI_1WX5p0d2",
      "_index": ".fs-logs-app-default-2025.10.31-000002",
      "_primary_term": 1,
      "_seq_no": 0,
      "_shards": {
        "failed": 0,
        "successful": 1,
        "total": 2
      },
      "_version": 1,
      "failure_store": "used",
      "forced_refresh": true,
      "result": "created"
    },
    "statusCode": 201
  }
}